modules/top-level/plugins/combine-plugins.nix

```
@@ -33,9 +33,8 @@ let
   optPlugins = removeDeps partitionedOptStartPlugins.right;
 
   # Test if plugin shouldn't be included in plugin pack
-  isStandalone =
-    p:
-    builtins.elem p.plugin standalonePlugins || builtins.elem (lib.getName p.plugin) standalonePlugins;
+  standaloneNames = map (p: if lib.isDerivation p then lib.getName p else p) standalonePlugins;
```

Would this be clearer and/or more robust:

```suggestion
  standaloneNames = map (p: if builtins.isString p then p else lib.getName p) standalonePlugins;
```
?

---

OK. Let's do this instead.